pkg/registry/populator_test.go

```diff
@@ -333,6 +339,82 @@ func TestImageLoading(t *testing.T) {
 	}
 }
 
+func TestQuerierForDependencies(t *testing.T) {
+	logrus.SetLevel(logrus.DebugLevel)
+	db, cleanup := CreateTestDb(t)
+	defer cleanup()
+
+	store, err := createAndPopulateDB(db)
+	require.NoError(t, err)
+
+	expectedDependencies := []*api.Dependency{
+		{
+			Type:  "olm.package",
+			Value: `{"packageName":"testoperator","type":"olm.package","version":"\u003e 0.2.0"}`,
+		},
+		{
+			Type:  "olm.gvk",
+			Value: `{"group":"testapi.coreos.com","kind":"testapi","type":"olm.gvk","version":"v1"}`,
+		},
+	}
+
+	type operatorbundle struct {
+		name    string
+		version string
+		path    string
+	}
+
+	bundlesList := []operatorbundle{
+		{
+			name:    "etcdoperator.v0.9.2",
+			version: "0.9.2",
+			path:    "quay.io/test/etcd.0.9.2",
+		},
+		{
+			name:    "prometheusoperator.0.22.2",
+			version: "0.22.2",
+			path:    "quay.io/test/prometheus.0.22.2",
+		},
```

Comment:
Can you add test cases for bundles that you didn't add a dependencies.yaml for? 

We should be able to pick up existing api dependencies, since they should be migrated by the migration.